cmd/commands/autofix.go

```diff
@@ -4,18 +4,30 @@ import (
 	"io"
 	"os"
 
+	"github.com/Shopify/kubeaudit/auditors/all"
+	"github.com/Shopify/kubeaudit/config"
 	log "github.com/sirupsen/logrus"
 	"github.com/spf13/cobra"
 )
 
 var autofixConfig struct {
-	outFile string
+	outFile             string
+	kubeauditConfigFile string
 }
 
 func autofix(cmd *cobra.Command, args []string) {
-	report := getReport()
+	conf := loadKubeAuditConfigFromFile(autofixConfig.kubeauditConfigFile)
+
+	conf = setConfigFromFlags(cmd, conf)
+
+	allAuditors, err := all.Auditors(conf)
```

Comment:
should I just call it `auditors`? or `configEnabledAuditors`?

Reply:
Auditors makes sense to me! This code looks like it's based on the `all` command so I think we should change it there too!